infrastructure/scripts/test_gpt5_api_access.py

```
@@ -0,0 +1,309 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)